### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by rakkuma**
Updating Change Log to include congestion control PR in  3.7.0-preview release

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## <a name="3.7.0-preview"/> [3.7.0-preview](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.7.0-preview) - 2020-02-25
 
+- [#1074](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/1074) Bulk API congestion control
```
Comment (kirankumarkolli) [inline changelog.md]:
```suggestion
- [#1074](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/1074) Adaptive throttling for bulk processing
```

Comment (j82w) [reply changelog.md]: @kirankumarkolli what about this?
```suggestion
- [#1074](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/1074) Bulk adaptive throttling
```

Comment (rakkuma) [reply changelog.md]: Is it a good idea to put throttling word in Changelog? And also Congestion control is widely used in TCP context.

Comment (ealsur) [reply changelog.md]: I like congestion control better. Adaptive throttling sounds like the library is the source of the throttling?

Comment (kirankumarkolli) [reply changelog.md]: Okey lets starts with congestion control

Comment (ealsur) [reply changelog.md]: @rakkuma can you do one more conflict resolve and go for congestion control?

Comment (rakkuma) [reply changelog.md]: Ack done.

**Pull request merged**